Tests/SentryTests/SentryCrash/SentryCrashJSONCodec_Tests.m

```diff
@@ -463,6 +463,55 @@ - (void)testSerializeDeserializeDictionaryMultipleEntries
     XCTAssertEqualObjects(result, original, @"");
 }
 
+- (void)testSerializeDeserializeDictionaryIntKeys
+{
+    NSString *expectedJson = @"{\"1\":\"Value\",\"2\":1000,\"3\":true}";
+    NSDictionary *expected = @{ @"1" : @"Value", @"2" : @1000, @"3" : @YES };
+    NSDictionary *original = @{ @1 : @"Value", @3 : @YES, @2 : @1000 };
+
+    [self testSerializeDeserializeDictionaryWith:original
+                                     andExpected:expected
+                                 andExpectedJSON:expectedJson];
+}
+
+- (void)testSerializeDeserializeDictionaryWith:(NSDictionary *)original
+                                   andExpected:(NSDictionary *)expected
+                               andExpectedJSON:(NSString *)expectedJson
+{
+    NSError *error = (NSError *)self;
+    NSString *jsonString = toString([SentryCrashJSONCodec encode:original
+                                                         options:SentryCrashJSONEncodeOptionSorted
+                                                           error:&error]);
+    XCTAssertNotNil(jsonString);
+    XCTAssertNil(error);
```

Comment:
L: is it actually possible to return a valid String and still have an error?

Reply:
I don't think so, but all the other tests are also written this way. So I would like to keep it for now.